### DISCUSSION TITLE
Cannot install spacy on Python 3.10.0 (64 bit) Windows 10

### DISCUSSION THREAD

**Discussion by iamhems** [explosion/spaCy#9417] [Help: Installation] [answered]
Hi All,

I'm struggling to install spaCy on my machine with pip using the documentation. I'm running windows 10 and Python 3.10.0 (64bit) and trying to install spaCy 3.1.3. Over the passed two days I've installed and unistalled python several times from python.org using their windows installed (not from windows store) with and without the PATH options and still receive the same result.

I also created fresh envirionment and activated it, yet get the same error. 

finally, i tried to install with ...pip install spacy --no-cache-dir 

see results below .... 

c:\dev\Automagixa\env>Scripts\activate

(env) c:\dev\Automagixa\env>pip install spacy --no-cache-dir
Collecting spacy
  Downloading spacy-3.1.3.tar.gz (1.0 MB)
     |████████████████████████████████| 1.0 MB 6.4 MB/s
  Installing build dependencies ... error
  ERROR: Command errored out with exit status 1:
   command: 'C:\dev\Automagixa\env\Scripts\python.exe' 'C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip' install --ignore-installed --no-user --prefix 'C:\Users\Admin\AppData\Local\Temp\pip-build-env-38zxllgf\overlay' --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- setuptools 'cython>=0.25,<3.0' 'cymem>=2.0.2,<2.1.0' 'preshed>=3.0.2,<3.1.0' 'murmurhash>=0.28.0,<1.1.0' 'thinc>=8.0.10,<8.1.0' 'blis>=0.4.0,<0.8.0' pathy 'numpy>=1.15.0'
       cwd: None
  Complete output (265 lines):
  Collecting setuptools
    Using cached setuptools-58.2.0-py3-none-any.whl (946 kB)
  Collecting cython<3.0,>=0.25
    Using cached Cython-0.29.24-py2.py3-none-any.whl (979 kB)
  Collecting cymem<2.1.0,>=2.0.2
    Using cached cymem-2.0.5.tar.gz (9.2 kB)
    Installing build dependencies: started
    Installing build dependencies: finished with status 'done'
    Getting requirements to build wheel: started
    Getting requirements to build wheel: finished with status 'done'
    Installing backend dependencies: started
    Installing backend dependencies: finished with status 'done'
      Preparing wheel metadata: started
      Preparing wheel metadata: finished with status 'done'
  Collecting preshed<3.1.0,>=3.0.2
    Using cached preshed-3.0.5.tar.gz (14 kB)
    Installing build dependencies: started
    Installing build dependencies: finished with status 'error'
    ERROR: Command errored out with exit status 1:
     command: 'C:\dev\Automagixa\env\Scripts\python.exe' 'C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip' install --ignore-installed --no-user --prefix 'C:\Users\Admin\AppData\Local\Temp\pip-build-env-q2571x9a\overlay' --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- setuptools 'cython>=0.28' 'cymem>=2.0.2,<2.1.0' 'murmurhash>=0.28.0,<1.1.0'
         cwd: None
    Complete output (82 lines):
    Collecting setuptools
      Using cached setuptools-58.2.0-py3-none-any.whl (946 kB)
    Collecting cython>=0.28
      Using cached Cython-0.29.24-py2.py3-none-any.whl (979 kB)
    Collecting cymem<2.1.0,>=2.0.2
      Using cached cymem-2.0.5.tar.gz (9.2 kB)
      Installing build dependencies: started
      Installing build dependencies: finished with status 'done'
      Getting requirements to build wheel: started
      Getting requirements to build wheel: finished with status 'done'
      Installing backend dependencies: started
      Installing backend dependencies: finished with status 'done'
        Preparing wheel metadata: started
        Preparing wheel metadata: finished with status 'done'
    Collecting murmurhash<1.1.0,>=0.28.0
      Using cached murmurhash-1.0.5.tar.gz (12 kB)
      Installing build dependencies: started
      Installing build dependencies: finished with status 'done'
      Getting requirements to build wheel: started
      Getting requirements to build wheel: finished with status 'done'
      Installing backend dependencies: started
      Installing backend dependencies: finished with status 'done'
        Preparing wheel metadata: started
        Preparing wheel metadata: finished with status 'done'
    Building wheels for collected packages: cymem, murmurhash
      Building wheel for cymem (PEP 517): started
      Building wheel for cymem (PEP 517): finished with status 'error'
      ERROR: Command errored out with exit status 1:
       command: 'C:\dev\Automagixa\env\Scripts\python.exe' 'C:\Users\Admin\AppData\Local\Temp\tmp84b9sskc_in_process.py' build_wheel 'C:\Users\Admin\AppData\Local\Temp\tmphp21j_sn'
           cwd: C:\Users\Admin\AppData\Local\Temp\pip-install-xtjw3jpb\cymem_5921e9a7e28d4bdd8831b2ebed9a2028
      Complete output (17 lines):
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build\lib.win-amd64-3.10
      creating build\lib.win-amd64-3.10\cymem
      copying cymem\about.py -> build\lib.win-amd64-3.10\cymem
      copying cymem\__init__.py -> build\lib.win-amd64-3.10\cymem
      package init file 'cymem\tests\__init__.py' not found (or not a regular file)
      creating build\lib.win-amd64-3.10\cymem\tests
      copying cymem\tests\test_import.py -> build\lib.win-amd64-3.10\cymem\tests
      copying cymem\cymem.pyx -> build\lib.win-amd64-3.10\cymem
      copying cymem\cymem.pxd -> build\lib.win-amd64-3.10\cymem
      copying cymem\__init__.pxd -> build\lib.win-amd64-3.10\cymem
      running build_ext
      building 'cymem.cymem' extension
      error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
      ----------------------------------------
      ERROR: Failed building wheel for cymem
      Building wheel for murmurhash (PEP 517): started
      Building wheel for murmurhash (PEP 517): finished with status 'error'
      ERROR: Command errored out with exit status 1:
       command: 'C:\dev\Automagixa\env\Scripts\python.exe' 'C:\Users\Admin\AppData\Local\Temp\tmpa5fmw2hl_in_process.py' build_wheel 'C:\Users\Admin\AppData\Local\Temp\tmpkkcfir3i'
           cwd: C:\Users\Admin\AppData\Local\Temp\pip-install-xtjw3jpb\murmurhash_ffa6fe90ac664084ad08e2e6bad1e399
      Complete output (22 lines):
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build\lib.win-amd64-3.10
      creating build\lib.win-amd64-3.10\murmurhash
      copying murmurhash\about.py -> build\lib.win-amd64-3.10\murmurhash
      copying murmurhash\__init__.py -> build\lib.win-amd64-3.10\murmurhash
      creating build\lib.win-amd64-3.10\murmurhash\tests
      copying murmurhash\tests\test_against_mmh3.py -> build\lib.win-amd64-3.10\murmurhash\tests
      copying murmurhash\tests\test_import.py -> build\lib.win-amd64-3.10\murmurhash\tests
      copying murmurhash\tests\__init__.py -> build\lib.win-amd64-3.10\murmurhash\tests
      copying murmurhash\mrmr.pyx -> build\lib.win-amd64-3.10\murmurhash
      copying murmurhash\mrmr.pxd -> build\lib.win-amd64-3.10\murmurhash
      copying murmurhash\__init__.pxd -> build\lib.win-amd64-3.10\murmurhash
      creating build\lib.win-amd64-3.10\murmurhash\include
      creating build\lib.win-amd64-3.10\murmurhash\include\murmurhash
      copying murmurhash\include\murmurhash\MurmurHash2.h -> build\lib.win-amd64-3.10\murmurhash\include\murmurhash
      copying murmurhash\include\murmurhash\MurmurHash3.h -> build\lib.win-amd64-3.10\murmurhash\include\murmurhash
      running build_ext
      building 'murmurhash.mrmr' extension
      error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
      ----------------------------------------
      ERROR: Failed building wheel for murmurhash
    Failed to build cymem murmurhash
    ERROR: Could not build wheels for cymem, murmurhash which use PEP 517 and cannot be installed directly
    ----------------------------------------
  WARNING: Discarding https://files.pythonhosted.org/packages/ee/87/cabd3dc3d7ebd9b62252faca25ec5f1fec627ea88ca7ffd2924d02e1516e/preshed-3.0.5.tar.gz#sha256=c6d3dba39ed5059aaf99767017b9568c75b2d0780c3481e204b1daecde00360e (from https://pypi.org/simple/preshed/). Command errored out with exit status 1: 'C:\dev\Automagixa\env\Scripts\python.exe' 'C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip' install --ignore-installed --no-user --prefix 'C:\Users\Admin\AppData\Local\Temp\pip-build-env-q2571x9a\overlay' --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- setuptools 'cython>=0.28' 'cymem>=2.0.2,<2.1.0' 'murmurhash>=0.28.0,<1.1.0' Check the logs for full command output.
    Using cached preshed-3.0.4.tar.gz (170 kB)
      ERROR: Command errored out with exit status 1:
       command: 'C:\dev\Automagixa\env\Scripts\python.exe' -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\Admin\\AppData\\Local\\Temp\\pip-install-vxjq3w0g\\preshed_a1f03b649283498a8199343ee2201262\\setup.py'"'"'; __file__='"'"'C:\\Users\\Admin\\AppData\\Local\\Temp\\pip-install-vxjq3w0g\\preshed_a1f03b649283498a8199343ee2201262\\setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base 'C:\Users\Admin\AppData\Local\Temp\pip-pip-egg-info-5e9s4su8'
           cwd: C:\Users\Admin\AppData\Local\Temp\pip-install-vxjq3w0g\preshed_a1f03b649283498a8199343ee2201262\
      Complete output (56 lines):
        ERROR: Command errored out with exit status 1:
         command: 'C:\dev\Automagixa\env\Scripts\python.exe' 'C:\dev\Automagixa\env\lib\site-packages\pip\_vendor\pep517\in_process\_in_process.py' build_wheel 'C:\Users\Admin\AppData\Local\Temp\tmpmnthll0r'
             cwd: C:\Users\Admin\AppData\Local\Temp\pip-wheel-4vn5gd2q\cymem_7e7dad8a2c9b4984b263e14bd7237d53
        Complete output (17 lines):
        running bdist_wheel
        running build
        running build_py
        creating build
        creating build\lib.win-amd64-3.10
        creating build\lib.win-amd64-3.10\cymem
        copying cymem\about.py -> build\lib.win-amd64-3.10\cymem
        copying cymem\__init__.py -> build\lib.win-amd64-3.10\cymem
        package init file 'cymem\tests\__init__.py' not found (or not a regular file)
        creating build\lib.win-amd64-3.10\cymem\tests
        copying cymem\tests\test_import.py -> build\lib.win-amd64-3.10\cymem\tests
        copying cymem\cymem.pyx -> build\lib.win-amd64-3.10\cymem
        copying cymem\cymem.pxd -> build\lib.win-amd64-3.10\cymem
        copying cymem\__init__.pxd -> build\lib.win-amd64-3.10\cymem
        running build_ext
        building 'cymem.cymem' extension
        error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
        ----------------------------------------
        ERROR: Failed building wheel for cymem
      ERROR: Failed to build one or more wheels
      Traceback (most recent call last):
        File "C:\dev\Automagixa\env\lib\site-packages\setuptools\installer.py", line 75, in fetch_build_egg
          subprocess.check_call(cmd)
        File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\subprocess.py", line 369, in check_call
          raise CalledProcessError(retcode, cmd)
      subprocess.CalledProcessError: Command '['C:\\dev\\Automagixa\\env\\Scripts\\python.exe', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', 'C:\\Users\\Admin\\AppData\\Local\\Temp\\tmpz8_e3epv', '--quiet', 'cymem<2.1.0,>=2.0.2']' returned non-zero exit status 1.

      The above exception was the direct cause of the following exception:

      Traceback (most recent call last):
        File "<string>", line 1, in <module>
        File "C:\Users\Admin\AppData\Local\Temp\pip-install-vxjq3w0g\preshed_a1f03b649283498a8199343ee2201262\setup.py", line 154, in <module>
          setup_package()
        File "C:\Users\Admin\AppData\Local\Temp\pip-install-vxjq3w0g\preshed_a1f03b649283498a8199343ee2201262\setup.py", line 116, in setup_package
          setup(
        File "C:\dev\Automagixa\env\lib\site-packages\setuptools\__init__.py", line 152, in setup
          _install_setup_requires(attrs)
        File "C:\dev\Automagixa\env\lib\site-packages\setuptools\__init__.py", line 147, in _install_setup_requires
          dist.fetch_build_eggs(dist.setup_requires)
        File "C:\dev\Automagixa\env\lib\site-packages\setuptools\dist.py", line 806, in fetch_build_eggs
          resolved_dists = pkg_resources.working_set.resolve(
        File "C:\dev\Automagixa\env\lib\site-packages\pkg_resources\__init__.py", line 766, in resolve
          dist = best[req.key] = env.best_match(
        File "C:\dev\Automagixa\env\lib\site-packages\pkg_resources\__init__.py", line 1051, in best_match
          return self.obtain(req, installer)
        File "C:\dev\Automagixa\env\lib\site-packages\pkg_resources\__init__.py", line 1063, in obtain
          return installer(requirement)
        File "C:\dev\Automagixa\env\lib\site-packages\setuptools\dist.py", line 877, in fetch_build_egg
          return fetch_build_egg(self, req)
        File "C:\dev\Automagixa\env\lib\site-packages\setuptools\installer.py", line 77, in fetch_build_egg
          raise DistutilsError(str(e)) from e
      distutils.errors.DistutilsError: Command '['C:\\dev\\Automagixa\\env\\Scripts\\python.exe', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', 'C:\\Users\\Admin\\AppData\\Local\\Temp\\tmpz8_e3epv', '--quiet', 'cymem<2.1.0,>=2.0.2']' returned non-zero exit status 1.
      ----------------------------------------
  WARNING: Discarding https://files.pythonhosted.org/packages/8a/25/73d38655125d46a543656c0f025b3394a2ee010af3d0c46ed75d554bc1a9/preshed-3.0.4.tar.gz#sha256=13a779205d55ce323976ac06df597f9ec2d6f0563ebcf5652176cf4520c7d540 (from https://pypi.org/simple/preshed/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
    Using cached preshed-3.0.3.tar.gz (170 kB)
  Collecting murmurhash<1.1.0,>=0.28.0
    Using cached murmurhash-1.0.5.tar.gz (12 kB)
    Installing build dependencies: started
    Installing build dependencies: finished with status 'done'
    Getting requirements to build wheel: started
    Getting requirements to build wheel: finished with status 'done'
    Installing backend dependencies: started
    Installing backend dependencies: finished with status 'done'
      Preparing wheel metadata: started
      Preparing wheel metadata: finished with status 'done'
  Collecting thinc<8.1.0,>=8.0.10
    Using cached thinc-8.0.10.tar.gz (172 kB)
    Installing build dependencies: started
    Installing build dependencies: finished with status 'error'
    ERROR: Command errored out with exit status 2:
     command: 'C:\dev\Automagixa\env\Scripts\python.exe' 'C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip' install --ignore-installed --no-user --prefix 'C:\Users\Admin\AppData\Local\Temp\pip-build-env-19gmgwdz\overlay' --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- setuptools 'cython>=0.25,<3.0' 'murmurhash>=0.28.0,<1.1.0' 'cymem>=2.0.2,<2.1.0' 'preshed>=3.0.2,<3.1.0' 'blis>=0.4.0,<0.8.0' 'numpy>=1.15.0'
         cwd: None
    Complete output (73 lines):
    Collecting setuptools
      Using cached setuptools-58.2.0-py3-none-any.whl (946 kB)
    Collecting cython<3.0,>=0.25
      Using cached Cython-0.29.24-py2.py3-none-any.whl (979 kB)
    Collecting murmurhash<1.1.0,>=0.28.0
      Using cached murmurhash-1.0.5.tar.gz (12 kB)
      Installing build dependencies: started
      Installing build dependencies: finished with status 'done'
      Getting requirements to build wheel: started
      Getting requirements to build wheel: finished with status 'done'
      Installing backend dependencies: started
      Installing backend dependencies: finished with status 'done'
        Preparing wheel metadata: started
        Preparing wheel metadata: finished with status 'done'
    Collecting cymem<2.1.0,>=2.0.2
      Using cached cymem-2.0.5.tar.gz (9.2 kB)
      Installing build dependencies: started
      Installing build dependencies: finished with status 'done'
      Getting requirements to build wheel: started
      Getting requirements to build wheel: finished with status 'done'
      Installing backend dependencies: started
      Installing backend dependencies: finished with status 'done'
        Preparing wheel metadata: started
        Preparing wheel metadata: finished with status 'done'
    Collecting preshed<3.1.0,>=3.0.2
      Using cached preshed-3.0.5.tar.gz (14 kB)
    ERROR: Exception:
    Traceback (most recent call last):
      File "C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip\_internal\cli\base_command.py", line 173, in _main
        status = self.run(options, args)
      File "C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip\_internal\cli\req_command.py", line 203, in wrapper
        return func(self, options, args)
      File "C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip\_internal\commands\install.py", line 315, in run
        requirement_set = resolver.resolve(
      File "C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip\_internal\resolution\resolvelib\resolver.py", line 94, in resolve
        result = self._result = resolver.resolve(
      File "C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip\_vendor\resolvelib\resolvers.py", line 472, in resolve
        state = resolution.resolve(requirements, max_rounds=max_rounds)
      File "C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip\_vendor\resolvelib\resolvers.py", line 341, in resolve
        self._add_to_criteria(self.state.criteria, r, parent=None)
      File "C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip\_vendor\resolvelib\resolvers.py", line 172, in _add_to_criteria
        if not criterion.candidates:
      File "C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip\_vendor\resolvelib\structs.py", line 151, in __bool__
        return bool(self._sequence)
      File "C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip\_internal\resolution\resolvelib\found_candidates.py", line 140, in __bool__
        return any(self)
      File "C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip\_internal\resolution\resolvelib\found_candidates.py", line 128, in <genexpr>
        return (c for c in iterator if id(c) not in self._incompatible_ids)
      File "C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip\_internal\resolution\resolvelib\found_candidates.py", line 32, in _iter_built
        candidate = func()
      File "C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip\_internal\resolution\resolvelib\factory.py", line 204, in _make_candidate_from_link
        self._link_candidate_cache[link] = LinkCandidate(
      File "C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip\_internal\resolution\resolvelib\candidates.py", line 295, in __init__
        super().__init__(
      File "C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip\_internal\resolution\resolvelib\candidates.py", line 156, in __init__
        self.dist = self._prepare()
      File "C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip\_internal\resolution\resolvelib\candidates.py", line 227, in _prepare
        dist = self._prepare_distribution()
      File "C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip\_internal\resolution\resolvelib\candidates.py", line 305, in _prepare_distribution
        return self._factory.preparer.prepare_linked_requirement(
      File "C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip\_internal\operations\prepare.py", line 508, in prepare_linked_requirement
        return self._prepare_linked_requirement(req, parallel_builds)
      File "C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip\_internal\operations\prepare.py", line 570, in _prepare_linked_requirement
        dist = _get_prepared_distribution(
      File "C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip\_internal\operations\prepare.py", line 59, in _get_prepared_distribution
        with req_tracker.track(req):
      File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\contextlib.py", line 135, in __enter__
        return next(self.gen)
      File "C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip\_internal\req\req_tracker.py", line 128, in track
        self.add(req)
      File "C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip\_internal\req\req_tracker.py", line 97, in add
        raise LookupError(message)
    LookupError: https://files.pythonhosted.org/packages/ee/87/cabd3dc3d7ebd9b62252faca25ec5f1fec627ea88ca7ffd2924d02e1516e/preshed-3.0.5.tar.gz#sha256=c6d3dba39ed5059aaf99767017b9568c75b2d0780c3481e204b1daecde00360e (from https://pypi.org/simple/preshed/) is already being built: preshed<3.1.0,>=3.0.2 from https://files.pythonhosted.org/packages/ee/87/cabd3dc3d7ebd9b62252faca25ec5f1fec627ea88ca7ffd2924d02e1516e/preshed-3.0.5.tar.gz#sha256=c6d3dba39ed5059aaf99767017b9568c75b2d0780c3481e204b1daecde00360e
    ----------------------------------------
  WARNING: Discarding https://files.pythonhosted.org/packages/44/6f/de85e350344a6255fd9acccdabfc60a622fe6f4ea5f628d7a39932f99e37/thinc-8.0.10.tar.gz#sha256=b5e4db8d24e6be8a5f1e4a17854772b79a2b560224519f50a21f3951ca24001f (from https://pypi.org/simple/thinc/) (requires-python:>=3.6). Command errored out with exit status 2: 'C:\dev\Automagixa\env\Scripts\python.exe' 'C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip' install --ignore-installed --no-user --prefix 'C:\Users\Admin\AppData\Local\Temp\pip-build-env-19gmgwdz\overlay' --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- setuptools 'cython>=0.25,<3.0' 'murmurhash>=0.28.0,<1.1.0' 'cymem>=2.0.2,<2.1.0' 'preshed>=3.0.2,<3.1.0' 'blis>=0.4.0,<0.8.0' 'numpy>=1.15.0' Check the logs for full command output.
  ERROR: Could not find a version that satisfies the requirement thinc<8.1.0,>=8.0.10 (from versions: 1.0, 1.1, 1.2, 1.3, 1.4, 1.5, 1.41, 1.42, 1.60, 1.61, 1.62, 1.63, 1.64, 1.65, 1.66, 1.67, 1.68, 1.69, 1.70, 1.71, 1.72, 1.73, 1.74, 1.75, 1.76, 2.0, 3.0, 3.1, 3.2, 3.3, 3.4.1, 4.0.0, 4.1.0, 4.2.0, 5.0.0, 5.0.1, 5.0.2, 5.0.3, 5.0.4, 5.0.5, 5.0.6, 5.0.7, 5.0.8, 6.0.0, 6.1.0, 6.1.1, 6.1.2, 6.1.3, 6.2.0, 6.3.0, 6.4.0, 6.5.0, 6.5.2, 6.6.0, 6.7.0, 6.7.1, 6.7.2, 6.7.3, 6.8.0, 6.8.1, 6.8.2, 6.9.0, 6.10.0, 6.10.1.dev0, 6.10.1, 6.10.2.dev0, 6.10.2.dev1, 6.10.2, 6.10.3.dev0, 6.10.3.dev1, 6.10.3, 6.10.4.dev0, 6.11.0.dev2, 6.11.1.dev0, 6.11.1.dev1, 6.11.1.dev2, 6.11.1.dev3, 6.11.1.dev4, 6.11.1.dev6, 6.11.1.dev7, 6.11.1.dev10, 6.11.1.dev11, 6.11.1.dev12, 6.11.1.dev13, 6.11.1.dev15, 6.11.1.dev16, 6.11.1.dev17, 6.11.1.dev18, 6.11.1.dev19, 6.11.1.dev20, 6.11.1, 6.11.2.dev0, 6.11.2, 6.11.3.dev1, 6.11.3.dev2, 6.12.0, 6.12.1, 7.0.0.dev0, 7.0.0.dev1, 7.0.0.dev2, 7.0.0.dev3, 7.0.0.dev4, 7.0.0.dev5, 7.0.0.dev6, 7.0.0.dev8, 7.0.0, 7.0.1.dev0, 7.0.1.dev1, 7.0.1.dev2, 7.0.1, 7.0.2, 7.0.3, 7.0.4.dev0, 7.0.4, 7.0.5.dev0, 7.0.5, 7.0.6, 7.0.7, 7.0.8, 7.1.0.dev0, 7.1.0, 7.1.1, 7.2.0.dev3, 7.2.0, 7.3.0.dev0, 7.3.0, 7.3.1, 7.4.0.dev0, 7.4.0.dev1, 7.4.0.dev2, 7.4.0, 7.4.1, 7.4.2, 7.4.3, 7.4.4, 7.4.5, 8.0.0.dev0, 8.0.0.dev2, 8.0.0.dev4, 8.0.0a0, 8.0.0a1, 8.0.0a2, 8.0.0a3, 8.0.0a6, 8.0.0a8, 8.0.0a9, 8.0.0a11, 8.0.0a12, 8.0.0a13, 8.0.0a14, 8.0.0a16, 8.0.0a17, 8.0.0a18, 8.0.0a19, 8.0.0a20, 8.0.0a21, 8.0.0a22, 8.0.0a23, 8.0.0a24, 8.0.0a25, 8.0.0a26, 8.0.0a27, 8.0.0a28, 8.0.0a29, 8.0.0a30, 8.0.0a31, 8.0.0a32, 8.0.0a33, 8.0.0a34, 8.0.0a35, 8.0.0a36, 8.0.0a40, 8.0.0a41, 8.0.0a42, 8.0.0a43, 8.0.0a44, 8.0.0rc0, 8.0.0rc1, 8.0.0rc2, 8.0.0rc3, 8.0.0rc4, 8.0.0rc5, 8.0.0rc6.dev0, 8.0.0rc6, 8.0.0, 8.0.1, 8.0.2, 8.0.3, 8.0.4, 8.0.5, 8.0.6, 8.0.7, 8.0.8, 8.0.9, 8.0.10)
  ERROR: No matching distribution found for thinc<8.1.0,>=8.0.10
  ----------------------------------------
WARNING: Discarding https://files.pythonhosted.org/packages/54/c9/95f5a8db73f8ab05b6dbf2a1727d8d77daedf7edfec469af59f57a08d8e7/spacy-3.1.3.tar.gz#sha256=58084e64a27997192ea48f18abb30ec148c5bbe79d04d17ba4d2fc262100c2a2 (from https://pypi.org/simple/spacy/) (requires-python:>=3.6). Command errored out with exit status 1: 'C:\dev\Automagixa\env\Scripts\python.exe' 'C:\Users\Admin\AppData\Local\Temp\pip-standalone-pip-9xmhv8ey\__env_pip__.zip\pip' install --ignore-installed --no-user --prefix 'C:\Users\Admin\AppData\Local\Temp\pip-build-env-38zxllgf\overlay' --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- setuptools 'cython>=0.25,<3.0' 'cymem>=2.0.2,<2.1.0' 'preshed>=3.0.2,<3.1.0' 'murmurhash>=0.28.0,<1.1.0' 'thinc>=8.0.10,<8.1.0' 'blis>=0.4.0,<0.8.0' pathy 'numpy>=1.15.0' Check the logs for full command output.

**Comment by adrianeboyd** [comment, marked as answer]
Hi, please use python 3.9 for now. Our build/CI infrastructure doesn't support python 3.10 yet, so it may be a month or so before binary wheels are available. A related issue: #9391

**Comment by svlandeg** [comment]
Please follow https://github.com/explosion/spaCy/discussions/9418 for updates on this.
